core/sql/cli/Cli.cpp

```diff
@@ -10599,7 +10599,10 @@ static Lng32 SeqGenCliInterfaceUpdAndValidateMulti(
       
       numTries++;
       
-      DELAY(100 + numTries*25);
+      if( 100 + numTries*25 < 1000)   //MAX is 1 second
+          DELAY(100 + numTries*25);
+      else
+          DELAY(1000);
```

Comment:
You might want to consider adding some small random amount to this maximum delay. Otherwise you might get a bunch of processes in lock step, all retrying at the same time.

Reply:
Yes, this is a good suggestion